apps/payments/next/postcss.config.js

```diff
@@ -2,6 +2,7 @@ const { join } = require('path');
 
 module.exports = {
   plugins: {
+    'postcss-import': {},
```

Comment:
I still see that error when trying to load localhost:3035. This can  be removed, as the error is actually coming from shared-styles and not payments-next.

Reply:
This actually corrected the error for me when building. Are you seeing a different error?